### PR TITLE
Replaces the UGH library with soldering code that uses the new modules

### PR DESCRIPTION
**Pull request by matthias-ronge**

### CODE REVIEW

**Comment by Kathrin-Huber** [comment]
please resolve conflicts

File: Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyContentFileHelper.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.kitodo.api.dataformat.mets.FLocatXmlElementAccessInterface;
+import org.kitodo.api.ugh.ContentFileInterface;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.dataformat.MetsService;
+
+/**
+ * Connects a legacy content file to a media file. This is a soldering class to
+ * keep legacy code operational which is about to be removed. Do not use this
+ * class.
+ */
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyContentFileHelper.java]: remove-todo is missing here

File: Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyRomanNumeralHelper.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+package org.kitodo.production.helper.metadata.legacytypeimplementations;
+
+import org.kitodo.api.ugh.RomanNumeralInterface;
+import org.kitodo.production.helper.metadata.RomanNumeral;
+
+/**
+ * Connects the legacy Roman numeral to a Roman numeral implementation. This is
+ * a soldering class to keep legacy code operational which is about to be
+ * removed. Do not use this class.
+ */
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyRomanNumeralHelper.java]: remove-todo is missing

**Comment by matthias-ronge** [comment]
Because the classes cannot be immediately removed, at least not as long as the logic of the metadata editors has been re-implemented on the new basis—and I suppose that will take a while. Only all dead methods can be removed promptly. Accordingly, I have marked the methods.

File: Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyRomanNumeralHelper.java (hunk 2 above)
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyRomanNumeralHelper.java]: The class can not be removed before all places in the code where it is used have been re-implemented.

File: Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyContentFileHelper.java (hunk 1 above)
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyContentFileHelper.java]: The class cannot be removed before all places in the code where it is used have been re-implemented.

**Pull request merged**